### ISSUE TITLE
bug introduced after in memory storing for GMAO/MERRA

### ISSUE THREAD

**Issue by dbekaert** [dbekaert/RAiDER#177]
@leiyangleon  the last PR on GMAO believe it introduced an issue for the filename that is expected.
It was not flagged as the "osr" issue occurred before.
PR is merged so tracking here for reference

```
    def test_getWMFilename():
        import datetime
        f = getWMFilename(
            'ERA5',
            datetime.datetime(2020, 1, 1, 0, 0, 0),
            'test_out_loc'
        )
        assert f[0]
>       assert f[1] == os.path.join(
            'test_out_loc',
            'ERA5_2020_01_01_T00_00_00.nc'
        )
E       AssertionError: assert 'test_out_loc..._T00_00_00.h5' == 'test_out_loc..._T00_00_00.nc'
E         - test_out_loc/ERA5_2020_01_01_T00_00_00.nc
E         ?                                        ^^
E         + test_out_loc/ERA5_2020_01_01_T00_00_00.h5
E         ?                                        ^^
```

**Comment by dbekaert** [comment]
Simple fix:

```
    if weather_model_name is 'GMAO' or weather_model_name is 'MERRA2':
```

Added user #179

**Comment by dbekaert** [comment]
closed per #179

**Comment by leiyangleon** [comment]
@dbekaert sorry my bad. Good catch!😊
